5:
  shoulder_pan_joint: -1.79734006
  shoulder_lift_joint: -0.75991636
  elbow_joint: 1.4489723
  wrist_1_joint: -3.8833576
  wrist_2_joint: -1.3446017
  wrist_3_joint: 0.01186824
